### ISSUE TITLE
Graph does not show

### ISSUE THREAD

**Issue by jarekj9** [rwbrockhoff/chartjsdemo#3]
At present, 'git clone' and 'npm run start' will display a page without graph. Perhaps something has changed in libs.

**Comment by michael-romero** [comment]
Yup, it's no longer working. Unfortunate as I was going to use this as a learning resource :(
